e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -41,148 +41,507 @@ public ProvisioningE2ETests()
             _listener = TestConfig.StartEventListener();
         }
 
-        public enum X509EnrollmentType
+        public enum EnrollmentType
         {
             Individual,
             Group
         }
 
+        public enum TransportProtocol
```

Comment:
>TransportProtocol [](start = 20, length = 17)

Why not reuse the DeviceClient transport protocol enum?

Reply:
That transport protocol enum includes WS implementations, but these tests are written to handle the WS case differently. It would take a good chunk of refactoring to use that enum, so I created one that is restricted to the non WS implementations for now.